src/server/boot/03-partioid-login.js

```diff
@@ -17,7 +17,9 @@ export default function (app) {
   const partioIdEntryPoint = `https://${partioIDRemoteName}.partio.fi/simplesaml/saml2/idp/SSOService.php`
   const partioIdLogoutUrl = `https://${partioIDRemoteName}.partio.fi/simplesaml/saml2/idp/SingleLogoutService.php`
   const partioIdCertificate = fs.readFileSync(
-    path.resolve(`./certs/partioid/${partioIDRemoteName}.crt`),
+    path.resolve(
+      `${__dirname}/../../../certs/partioid/${partioIDRemoteName}.crt`,
```

Comment:
is this required to make the tests work? In general I prefer not using `__dirname`, since it makes the path fragile to refactorings (if the call site is moved to a different file or the file is moved to a different directory)

Reply:
Unfortunately yes - it's somehow related to the Cypress TypeScript config having a different current working directory than the server. Would be nice to come up with a nicer solution but I'm "takki auki" here so to speak :(